mtags/src/main/scala-3/scala/meta/internal/pc/PcDefinitionProvider.scala

```diff
@@ -130,13 +130,15 @@ class PcDefinitionProvider(
       case (imp: Import) :: _ =>
         importedSymbols(imp, _.span.contains(pos.span))
 
-      // For constructor calls, return the `<init>` that was selected
-      case _ :: (_: New) :: (select: Select) :: _ =>
-        List(select.symbol)
-
       // wildcard param
       case head :: _ if (head.symbol.is(Param) && head.symbol.is(Synthetic)) =>
-        Nil
+        List(head.symbol)
+
+      case (head @ Select(target, name)) :: _
+          if head.symbol.is(Synthetic) && name == StdNames.nme.apply =>
+        val sym = target.symbol
```

Comment:
Could you add a comment which case is handled by this match for future us?

Reply:
it's for synthetic apply method in case-class